forte_wrapper/cliner/__init__.py

```diff
@@ -0,0 +1 @@
+from forte_wrapper.cliner.Cliner import *
```

Comment:
Import * will import the classes defined in the `__all__` section. I didn't see the `__all__` definition in the file Cliner, you might want to add that to make this import useful